modules/040-node-manager/crds/sshcredentials.yaml

```diff
@@ -38,17 +39,27 @@ spec:
               description: SSHCredentialsSpec defines the desired state of SSHCredentials
               properties:
                 privateSSHKey:
+                  description: |
+                    Private SSH key in PEM format encoded as base64 string.
                   type: string
                 sshExtraArgs:
+                  description: |
+                    A list of additional arguments to pass to the openssh command.
```

Comment:
I think there is should be some examples

Reply:
@artbin can you add some examples in the comment?

Reply:
done